pkg/config/config_template.yaml

```diff
@@ -546,7 +546,21 @@ api_key:
     #
     # ad_identifier: snmp
 
+{{- if .Profiling -}}
+## @param profiling - custom object - optional
+## Enter specific configurations for profiling.
+## Uncomment this parameter and the one below to enable profiling.
+#
+# profiling:
+#
+  ## @param enabled - boolean - optional - default: false
+  ## Enable profiling for the Agent process.
+  #
+  # enabled: false
+
+{{ end }}
```

Comment:
Should we include this since the flag is referred to as hidden elsewhere    ? Or at least include the same disclaimer

Reply:
this should indeed be removed, my bad.

Reply:
So what I ended up doing was adding a disclaimer, but also removing the section from the generated `datadog.yaml`, that way we already have that ready and in there.